integrations/express/middleware.ts

```diff
@@ -139,6 +139,9 @@ export function afterMiddleware(keploy: Keploy, req: Request, res: Response) {
     return;
   }
 
+  if (process.env.KEPLOY_MODE=="test") {
```

Comment:
Please use MODE_TEST constant intead of "test" here and use "equality without type coersion”.

Reply:
Fixed.

Reply:
LGTM. Hey @shreyanshshah27, please add link to this PR in [GSOC/2023](https://github.com/keploy/gsoc/blob/main/2023/README.md#3-keploy-typescriptjavascript-sdk) Task List.